src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -132,6 +165,9 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, location
         try:
             resourceClient.resource_groups.create_or_update(resource_group_name, resource_group_params)
         except Exception as e:
+            telemetry.set_user_fault()
+            telemetry.set_exception(exception=e, fault_type='resource-group-creation-error',
```

Comment:
Question: Why do we think we should mark this as user fault?

Reply:
I cannot think of a reason for changing our code if the resource group creation fails. It probably should be permission. But since it could also be connectivity issue, i also think we should not set user fault here